### ISSUE TITLE
Speak Screen does not work on iOS

### ISSUE THREAD

**Issue by eseidelGoogle** [flutter/flutter#11652]
I you turn on Speak Screen and attempt to speak the screen (pull down with two fingers from the top) in a Flutter app it just says "no speakable content could be found on the screen".

If you turn on VoiceOver however there are lots of speakable things.
